ios-settings.gradle

```diff
@@ -0,0 +1 @@
+include ':ios-combined'
```

Comment:
Is this required by default?  ref: https://github.com/DroidKaigi/conference-app-2020/pull/447/files#diff-900def4b7f2013a4665d633b21528b04R3

Reply:
I added this file. If Android Studio loads this. The duplicate file error will be occurring 😭